crates/esplora/src/blocking_ext.rs

```diff
@@ -46,7 +46,7 @@ pub trait EsploraExt {
     /// transactions. `parallel_requests` specifies the max number of HTTP requests to make in
     /// parallel.
     #[allow(clippy::result_large_err)]
-    fn update_tx_graph<K: Ord + Clone>(
+    fn sync_with_keychain<K: Ord + Clone>(
```

Comment:
I think this should be called `scan_with_keychain`

Reply:
This has been resolved